### PR TITLE
Allow temperature, speed and flow control when printing

### PR DESCRIPTION
**Pull request by Lowentwickler**
Snímek obrazovky z 2022-08-11 15-37-54

### CODE REVIEW

**Comment by JakubSilar3D** [comment]
image
Just a side note: these is a different styling for similar situations in SLA PL

**Comment by ondratu** [comment]
> image Just a side note: these is a different styling for similar situations in SLA PL

Yes, we solve next time.

File: src/printer/components/control.js [hunk 1]
```diff
@@ -57,7 +57,14 @@ function initDisableSteppersBtn() {
 
 function updateButtons(state) {
   const disabled = state.flags.printing || state.flags.pausing || state.flags.paused;
-  const whitelist = ["move-step", "extrude-retract-step"];
+  const whitelist = [
+    "move-step",
```
Comment (ondratu) [inline src/printer/components/control.js:61]: This have to be divided:
**printing**:
  * flowrate
  * bed
  * nozzle
  * spped
  
**paused**:
  * all from printing plus
  * move-step
  * extrude-retract-step - i hope that this is for both operations, extrude and retract

**Review by ondratu — CHANGES_REQUESTED** [review]

File: src/printer/components/control.js (hunk 1 above)
Comment (Lowentwickler) [reply src/printer/components/control.js]: Paused state
Snímek obrazovky z 2022-08-12 11-21-08

**Review by ondratu — APPROVED** [review]

**Pull request merged**